lib/python/qmk/cli/doctor/check.py

```diff
@@ -159,6 +159,6 @@ def check_git_repo():
     This is a decent enough indicator that the qmk_firmware directory is a
     proper Git repository, rather than a .zip download from GitHub.
     """
-    dot_git_dir = QMK_FIRMWARE / '.git'
+    dot_git = QMK_FIRMWARE / '.git'
 
-    return CheckStatus.OK if dot_git_dir.is_dir() else CheckStatus.WARNING
+    return CheckStatus.OK if dot_git.is_dir() or dot_git.is_file() else CheckStatus.WARNING
```

Comment:
Should we just use `.exists()` here so that symlinks can work too?

Reply:
Sure, I don't mind changing it to `.exists()`, but I've never seen a symlinked `.git` folder/directory.
The file version of `.git` is just created by `git submodule add`.